transport/grpc/dial.go

```diff
@@ -298,8 +332,14 @@ func dial(ctx context.Context, insecure bool, o *internal.DialSettings) (*grpc.C
 			if err != nil {
 				return nil, err
 			}
+
+			ts := creds.TokenSource
```

Comment:
This code will not be hit if the new auth lib is enabled. That that intentional here?

Reply:
Yes